plugins/source/aws/resources/services/ec2/eips.go

```diff
@@ -20,7 +20,7 @@ func Eips() *schema.Table {
 		Description: `https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_Address.html`,
 		Resolver:    fetchEc2Eips,
 		Multiplex:   client.ServiceAccountRegionMultiplexer(tableName, "ec2"),
-		Transform:   transformers.TransformWithStruct(&types.Address{}),
+		Transform:   transformers.TransformWithStruct(&types.Address{}, transformers.WithPrimaryKeys("AllocationId")),
 		Columns: []schema.Column{
 			client.DefaultAccountIDColumn(false),
 			client.DefaultRegionColumn(false),
```

Comment:
```suggestion
			client.DefaultAccountIDColumn(true),
			client.DefaultRegionColumn(true),
```

I'm not 100% sure that `AllocationId` is guaranteed to be unique across accounts and regions (at least I couldn't find AWS documentation that confirms this) so I would recommend also adding `account` and `region` as PKs so that records aren't accidentally overwritten.